src/docs/015_tasks/03_task_generatePDF.adoc

```diff
@@ -7,17 +7,35 @@ include::../_feedback.adoc[]
 
 == At a Glance
 
-image::ea/Manual/generatePDF.png[]
-
 == About This Task
 
 This task makes use of the https://asciidoctor.org/docs/convert-asciidoc-to-pdf/[asciidoctor-pdf] plugin to render your documents as pretty PDF files.
 Files are written to `build/pdf`.
 The PDF is generated directly from your AsciiDoc sources. There is no need for an intermediate format or other tools.
 The result looks more like a nicely rendered book than a print-to-PDF HTML page.
 
-NOTE: The asciidoctor-pdf plugin is still in alpha status, but the results are already quite good.
-If you want an alternative, use https://phantomjs.org/[PhantomJS] for your instance and script it.
+For a file to be rendered, it has to be configured in the `doctoolchainConfig.groovy` file.
+There you will find a section like the following:
+
+```
+inputFiles = [
+        [file: 'manual.adoc',       formats: ['html','pdf']],
+        /** inputFiles **/
+]
+```
+
+Add the files which you want to be rendered along with the format. 
+In this case `pdf`.
+
+.Hint
+[%collapsible]
+====
+Why do you need to configure the files to be rendered?
+
+Asciidoctor renders all `.adoc` files by default, no matter if these are the main documents or just chapters you want to include.
+In addition, it is quite likely that you only want to convert a few files to PDF.
```

Comment:
Most people only want to convert selected files to PDF, so that's why you need to configure which ones.